README.md

```diff
@@ -92,6 +92,7 @@ Like `awesome-developer-streams`? Reach out to [@bitandbang](https://twitter.com
 - [Kyle Shevlin](#kyle-shevlin) - **streaming:** React, JavaScript, Functional Programming, Web Development
 - [Luke Gorrie](#luke-gorrie) - **streaming:** C, Lua, RaptorJIT, R, Nix, Network drivers
 - [Mark Kraus](#mark-kraus) - **streaming** PowerShell, C#, Azure Functions, CI/CD Pipelines, OSS
+- [Marco Bruno](#marco-bruno) - **streaming** Programing, Javascript, Web Development, React, CSS, HTML
```

Comment:
Marco is alphabetically before Mark so could this (and the section details below) be moved above Mark Kraus?

Also, the English spelling of the first word in the description is Programming (with 2 m characters).

Reply:
Resolved 😄